app/models/Broadcast.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+/**
+ * Imports.
+ */
+const mongoose = require('mongoose');
+
+/**
+ * Schema.
+ */
+const broadcastSchema = new mongoose.Schema({
+
+  _id: { type: Number, index: true },
+  type: { type: String, enum: ['signup_prompt'] },
+  campaign: Number,
+  messages: {
+    signup_prompt_declined: String,
```

Comment:
Ok, will leave it this way. I'm now leaning toward renaming this as `prompt_declined`, to repurpose the property for the future `reportback_prompt` feature we'll need to close out #742